### DISCUSSION TITLE
How to modify the resolution | 怎么修改分辨率

### DISCUSSION THREAD

**Discussion by slvolf** [wxFormBuilder/wxFormBuilder#831] [Q&A]
It may be that I don't use the software deeply.
I can't find the option to zoom the control.
That is, the software cannot switch between 1080p and 720p.
Is it that I didn't find it, or I didn't find this setting, will there be this setting in the future?

可能是我对软件的使用不深，
我找不到缩放控件的选项，
即软件不能在1080p与720p间切换。
是我没有找到，还是没有找到这个设置，以后会有这个功能吗？

**Comment by sodevel** [comment]
I am sorry but i don't understand your question. Which control do you want to zoom? And what do you mean by software? The wxFB application itself or the GUI of your application you create with wxFB?

**Comment by slvolf** [comment]
The GUI of my application create with wxFB.
It doesn't have to be, but it's more convenient to have it.
I want it just because I used to code to modify the global size multiplier.
________________________________
发件人: Steffen Olszewski ***@***.***>
发送时间: 2024年7月26日 22:36
收件人: wxFormBuilder/wxFormBuilder ***@***.***>
抄送: aerbeiga ***@***.***>; Author ***@***.***>
主题: Re: [wxFormBuilder/wxFormBuilder] How to modify the resolution | 怎么修改分辨率 (Discussion #831)


I am sorry but i don't understand your question. Which control do you want to zoom? And what do you mean by software? The wxFB application itself or the GUI of your application you create with wxFB?

―

**Comment by sodevel** [comment]
wxFB is just a GUI builder for the wxWidgets framework, i am not aware that it includes such a zoom functionality.
